Formula/qt6.rb

```diff
@@ -0,0 +1,172 @@
+class Qt6 < Formula
+  desc "Cross-platform application and UI framework"
+  homepage "https://www.qt.io/"
+  url "https://download.qt.io/official_releases/qt/6.0/6.0.1/single/qt-everywhere-src-6.0.1.tar.xz"
+  sha256 "d13cfac103cd80b216cd2f73d0211dd6b1a1de2516911c89ce9c5ed14d9631a8"
+  license all_of: ["GFDL-1.3-only", "GPL-2.0-only", "GPL-3.0-only", "LGPL-2.1-only", "LGPL-3.0-only"]
+
+  depends_on "cmake" => :build
+  depends_on "ninja" => :build
+  depends_on xcode: :build
+
+  depends_on "assimp"
+  depends_on "dbus"
+  depends_on "double-conversion"
+  depends_on "freetype"
+  depends_on "glib"
+  depends_on "icu4c"
+  depends_on "jpeg"
+  depends_on "libb2"
+  depends_on "libpng"
+  depends_on "libproxy"
+  depends_on "llvm"
+  depends_on "pcre2"
+  depends_on "pkg-config"
```

Comment:
Not a build dependency?

Reply:
Just runtime dependency, qdoc use llvm as doc parser.

Reply:
I was referring to `pkg-config`.

Reply:
modified

Reply:
And for LLVM, can we not use the macOS LLVM?

Reply:
from official doc, it use some clang's lib

Reply:
There might be a opportunity to mark llvm as build dependency, because llvm is a bunch of static library, add some cmake flags could force it link against static lib of clang, not libclang.dylib